swift/modules/SwiftUtilities/Package.swift

```diff
@@ -30,20 +30,22 @@ let package = Package(
             name: "SwiftUtilities",
             dependencies: [
                 "Fakery"
-            ],
-            linkerSettings: [
-                .linkedLibrary("rt")    // Include librt for Dispatch to work.
             ]
         ),
         .testTarget(
             name: "SwiftUtilitiesTests",
             dependencies: [
                 "SwiftUtilities",
                 "Fakery"
-            ],
-            linkerSettings: [
-                .linkedLibrary("rt")    // Include librt for Dispatch to work.
             ]
         ),
     ]
 )
+
+#if !os(macOS) && !os(iOS) && !os(watchOS) && !os(tvOS)
+    // Include librt for Dispatch to work on Linux.
+for target in package.targets {
+    target.linkerSettings?.append(LinkerSetting.linkedLibrary("rt"))
+}
+
+#endif
```

Comment:
I haven't had to include `librt` on Linux in some time. I just removed this block and did a clean build and run successfully on Ubuntu.